consts/event_type.py

```diff
@@ -3,6 +3,7 @@ class EventType(object):
     DISTRICT = 1
     DISTRICT_CMP = 2
     CMP_DIVISION = 3
+    CMP_SUBDIVISION = 5
```

Comment:
Are there subdivisions now? I'm not sure the data model.

I think we should keep these enum lists in numerical order, so you can make sure you're not using a value already used.